### PR TITLE
Parity for namespace handling in secrets E2E

### PR DESCRIPTION
**Pull request by pmorie**
@zmerlynn PTAL

### CODE REVIEW

**Comment by pmorie** [comment]
kicked shippable

File: test/e2e/secrets.go [hunk 1]
```diff
@@ -29,15 +29,27 @@ import (
...
 	var c *client.Client
+	var ns string
 
 	BeforeEach(func() {
 		var err error
 		c, err = loadClient()
 		Expect(err).NotTo(HaveOccurred())
+		ns_, err := createTestingNS("secrets", c)
+		ns = ns_.Name
+		Expect(err).NotTo(HaveOccurred())
```
Comment (zmerlynn) [inline test/e2e/secrets.go]: expectNoError(err)

**Comment by pmorie** [comment]
@zmerlynn fixed

**Comment by zmerlynn** [comment]
LGTM

**Comment by zmerlynn** [comment]
Will merge on green.

**Comment by pmorie** [comment]
@zmerlynn Light is green, trap is clean

**Pull request merged**